lecture_2/homework/db/migrate/20190331172342_create_weapons.rb

```diff
@@ -0,0 +1,14 @@
+class CreateWeapons < ActiveRecord::Migration[5.2]
+  def change
+    create_table :weapons do |t|
+      t.string :weapon_name, null: false, unique: true
```

Comment:
Definiujesz tutaj kolumnę `weapon_name`, a w modelu walidujesz `name`. Zdaje się, że to przeoczenie, bo w schemie masz `name`, czyli kiedyś musiało być dobrze